core/SConscript.unix

```diff
@@ -761,6 +765,13 @@ def cargo_build():
     if DMA2D:
         features.append('dma2d')
 
+    if TREZOR_LANG == 'en':
+        features.append('lang_en')
+    elif TREZOR_LANG == 'cs':
+        features.append('lang_cs')
```

Comment:
```suggestion
    if TREZOR_LANG in ['en', 'cs']:
        features.append(f'lang_{TREZOR_LANG}')
```

Reply:
Done in [8ad8475](https://github.com/trezor/trezor-firmware/pull/3206/commits/8ad8475eea41f7b66cbcd2b95f969ed0677a65ec)